<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>My_contact_manager_Form</class>
 <widget class="QWidget" name="My_contact_manager_Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>800</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>410</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>410</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string notr="true">My contact manager</string>
  </property>
  <widget class="QWidget" name="centralwidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>409</width>
     <height>801</height>
    </rect>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>490</y>
      <width>391</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QPushButton" name="Clear_pushButton">
       <property name="text">
        <string notr="true">clear</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="Add_contact_pushButton">
       <property name="text">
        <string notr="true">add contact</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Edit_contact_pushButton">
       <property name="text">
        <string notr="true">edit contact</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Del_contact_pushButton">
       <property name="text">
        <string notr="true">del contact</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QScrollArea" name="Type_group_preview_scrollArea">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>49</y>
      <width>391</width>
      <height>432</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="widgetResizable">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>365</width>
       <height>430</height>
      </rect>
     </property>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>371</width>
        <height>421</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout"/>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>760</y>
      <width>391</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QCheckBox" name="Command_sender_checkBox">
       <property name="text">
        <string notr="true">cmd sender</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="Automatic_on_hook_checkBox">
       <property name="text">
        <string notr="true">automatic on-hook</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_7">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="Dial_pushButton">
       <property name="text">
        <string notr="true">dial</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="On_hook_pushButton">
       <property name="text">
        <string notr="true">on-hook</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTextEdit" name="Info_textEdit">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>530</y>
      <width>391</width>
      <height>222</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Help_pushButton">
    <property name="geometry">
     <rect>
      <x>353</x>
      <y>14</y>
      <width>20</width>
      <height>23</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>20</width>
      <height>23</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>20</width>
      <height>23</height>
     </size>
    </property>
    <property name="toolTip">
     <string notr="true"/>
    </property>
    <property name="text">
     <string notr="true">h</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Search_pushButton">
    <property name="geometry">
     <rect>
      <x>297</x>
      <y>14</y>
      <width>50</width>
      <height>23</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>50</width>
      <height>23</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>50</width>
      <height>23</height>
     </size>
    </property>
    <property name="text">
     <string notr="true">search</string>
    </property>
   </widget>
   <widget class="QPushButton" name="About_pushButton">
    <property name="geometry">
     <rect>
      <x>379</x>
      <y>14</y>
      <width>20</width>
      <height>23</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>20</width>
      <height>23</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>20</width>
      <height>23</height>
     </size>
    </property>
    <property name="text">
     <string notr="true">?</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
